packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -246,6 +265,8 @@ export class GarbageCollector implements IGarbageCollector {
     private readonly deleteTimeoutMs: number;
     // Map of node ids to their unreferenced state tracker.
     private readonly unreferencedNodesState: Map<string, UnreferencedStateTracker> = new Map();
+    // The timeout responsible for closing the container when the container runtime has expired
+    private readonly sessionExpiryTimeout?: ReturnType<typeof setTimeout>;
```

Comment:
This should probably be called `sessionExpiryTimer` since this is not a timeout but rather the running timer.